Screenbox/Pages/MainPage.xaml.cs

```diff
@@ -54,6 +57,10 @@ public MainPage()
             //Register a handler for when the window changes focus
             Window.Current.CoreWindow.Activated += CoreWindow_Activated;
 
+            //Register a handler for when the theme mode changes
+            _themeListener = new ThemeListener();
+            _themeListener.ThemeChanged += ThemeListener_OnThemeChanged;
```

Comment:
There is an `ActualThemeChanged` event in the `FrameworkElement` class. I think this listener is a legacy code.

https://learn.microsoft.com/en-us/uwp/api/windows.ui.xaml.frameworkelement.actualthemechanged?view=winrt-26100#windows-ui-xaml-frameworkelement-actualthemechanged

Reply:
Thanks, didn't knew about it.